tox.ini

```diff
@@ -97,5 +102,7 @@ whitelist_externals =
 setenv =
     PYTHONPATH = {envtmpdir}/mongo-python-driver
 commands =
-    git clone --depth 1 https://github.com/mongodb/mongo-python-driver.git {envtmpdir}/mongo-python-driver
+    pip install git+https://github.com/mongodb/mongo-python-driver.git@v3.12#egg=pymongo[encryption]
```

Comment:
This is incorrect. I don't think synchro can work like this... Your previous changes were correct for this... Something like:
```
git clone --depth 1 --branch v3.12 https://github.com/mongodb/mongo-python-driver.git {envtmpdir}/mongo-python-driver
```

Reply:
ok, I'll retest but the a lot of errors from synchro were poppiing up